index.d.ts

```diff
@@ -44,13 +49,14 @@ declare namespace Eris {
   type RequestMethod = "GET" | "PATCH" | "DELETE" | "POST" | "PUT";
 
   // Guild
-  type DefaultNotifications = 0 | 1;
-  type ExplicitContentFilter = 0 | 1 | 2;
-  type GuildFeatures = "ANIMATED_ICON" | "BANNER" | "COMMERCE" | "COMMUNITY" | "DISCOVERABLE" | "FEATURABLE" | "INVITE_SPLASH" | "MEMBER_VERIFICATION_GATE_ENABLED" | "NEWS" | "PARTNERED" | "PREVIEW_ENABLED" | "VANITY_URL" | "VERIFIED" | "VIP_REGIONS" | "WELCOME_SCREEN_ENABLED" | "TICKETED_EVENTS_ENABLED" | "MONETIZATION_ENABLED" | "MORE_STICKERS" | "THREE_DAY_THREAD_ARCHIVE" | "SEVEN_DAY_THREAD_ARCHIVE" | "PRIVATE_THREADS";
-  type NSFWLevel = 0 | 1 | 2 | 3;
+  type DefaultNotifications = (Constants["DefaultMessageNotificationLevel"])[keyof Constants["DefaultMessageNotificationLevel"]];
+  type ExplicitContentFilter = (Constants["ExplicitContentFilterLevel"])[keyof Constants["ExplicitContentFilterLevel"]];
+  type GuildFeatures = Constants["GuildFeatures"][number];
+  type NSFWLevel = (Constants["GuildNSFWLevel"])[keyof Constants["GuildNSFWLevel"]];
   type PossiblyUncachedGuild = Guild | Uncached;
-  type PremiumTier = 0 | 1 | 2 | 3;
-  type VerificationLevel = 0 | 1 | 2 | 3 | 4;
+  type PremiumTier = (Constants["PremiumTier"])[keyof Constants["PremiumTier"]];
+  type VerificationLevel = (Constants["VerificationLevel"])[keyof Constants["VerificationLevel"]];
+  type SystemChannelFlags = (Constants["SystemChannelFlags"])[keyof Constants["SystemChannelFlags"]];
```

Comment:
```suggestion
  type PremiumTier = Constants["PremiumTier"][keyof Constants["PremiumTier"]];
  type VerificationLevel = Constants["VerificationLevel"][keyof Constants["VerificationLevel"]];
  type SystemChannelFlags = Constants["SystemChannelFlags"][keyof Constants["SystemChannelFlags"]];
```
Almost forgot about here